cypress/integration/helloworld_test.ts

```diff
@@ -1,122 +1,91 @@
 describe('Hello world e2e', function () {
-    const newAppName = `e2e-HelloWorld-${new Date().getTime() - 1528151000000}`
-    const username = `UserName`
-    const action01 = 'Hello World!'
-    /**
-     * FEATURE: New Model
-     */
-    it('CREATE NEW MODEL with random name and verify name on application page', function () {
-      // Open application
-      cy.visit('http://localhost:5050')
-  
-      // Click the button to create app
-      cy
-        .get('[data-testid="apps-list-button-create-new"]')
-        .click()
-  
-      // Ensure that name input is focused
-      cy.focused()
-  
-      cy
-        .get('[data-testid="app-create-input-name"]')
-        .type(newAppName)
-  
-      // Click the submit button
-      cy
-        .get('[data-testid="app-create-button-submit"]')
-        .click()
-  
-      // Ensure app page displays new application title
-      cy
-        .get('[data-testid="app-index-title"]')
-        .should(el => {
-          expect(el).to.contain(newAppName)
-        })
+  const postfix = new Date().getTime() - 1528151000000
+  const modelName = `e2e-HelloWorld-${postfix}`
+  const entity01 = `UserName-${postfix}`
+  const action01 = `Hello World`
+  const trainmessage01 = `Hello`
+  const trainmessage02 = `Hi`
+
+  /** FEATURE: New Model */
+  it('CREATE NEW MODEL with random name and verify name on application page', function () {
+    var modelpage = require('../support/components/modelpage')
```

Comment:
All imports should be at the top of the page. Also, I believe you should be able to ES2015 syntax